### PR TITLE
778 Make publishing state available as fpp property. Remove unneeded database call

### PR DESCRIPTION
**Pull request by robbdavis**
778 Make publishing state available as fpp property. Remove unneeded database call

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core.module [hunk 1]
```diff
@@ -484,14 +484,22 @@ function ucgy_core_panels_pre_render($panels_display, $renderer)
...
+            ->execute()
+            ->fetchAll();
+
+          // Define published states as variables
+          $published    = $query[0]->fpp_publish;
+          $publish_on   = $query[0]->fpp_publish_on;
+          $unpublsih_on = $query[0]->fpp_unpublish_on;
+
+          // Make published states available as an fpp property.
+          $fpp->published_sate = $published;
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core.module]: Correct $fpp->published_sate to $fpp->published_state

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_fpp_scheduling/ucgy_core_fpp_scheduling.module [hunk 2]
```diff
@@ -968,28 +968,21 @@ function ucgy_core_fpp_scheduling_panels_pre_render($panels_display)
...
-            // Grab fpp scheduling info from the db
-            $query = db_select('fpp_scheduling', 'f')
-                ->fields('f', array('fpp_publish', 'fpp_publish_on', 'fpp_unpublish_on'))
-                ->condition('f.entity_id', $fpp->fpid, '=')
-                ->execute()
-                ->fetchAll();
-
             // If the fpp is published
-            if ($query[0]->fpp_publish == 1){
+            if ($fpp->published_sate == 1){
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_fpp_scheduling/ucgy_core_fpp_scheduling.module]: Correct $fpp->published_sate to $fpp->published_state

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_fpp_scheduling/ucgy_core_fpp_scheduling.module [hunk 3]
```diff
@@ -968,28 +968,21 @@ function ucgy_core_fpp_scheduling_panels_pre_render($panels_display)
...
 
                 // Create a fpp variable that will display a green checkmark via html
                 $fpp->is_published = '<div class="publishing-links-wrapper light-green-back"><p><span class="glyphicon glyphicon-ok"></span></p></div>';
             }
 
             // If the fpp is unpublished
-            if ($query[0]->fpp_publish == 0) {
+            if ($fpp->published_sate == 0) {
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_fpp_scheduling/ucgy_core_fpp_scheduling.module]: Correct $fpp->published_sate to $fpp->published_state

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_fpp_scheduling/ucgy_core_fpp_scheduling.module (hunk 2 above)
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_fpp_scheduling/ucgy_core_fpp_scheduling.module]: Fixed.

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_fpp_scheduling/ucgy_core_fpp_scheduling.module (hunk 3 above)
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_fpp_scheduling/ucgy_core_fpp_scheduling.module]: Fixed.

**Comment by robbdavis** [comment]
Fixed requested changes.

**Pull request merged**